runtime/doc/options.txt

```diff
@@ -6762,8 +6776,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 	displayed when 'verbosefile' is set.
 
 						*'viewdir'* *'vdir'*
-'viewdir' 'vdir'	string	(default for MS-DOS and Win32: "$VIM/vimfiles/view",
-				 for Unix: "~/.vim/view")
+'viewdir' 'vdir'	string	(default: "$XDG_DATA_HOME/nvim/view")
```

Comment:
@ZyX-I hmm, I seem to have missed the relevant discussion about placing swap files in DATA instead of CACHE (the only thing I found was my own remark, and your comment referring it, both linked above, which do not apply, as that was about not using RUNTIME). As to `viewdir` … view session files are *important*, yes, but *essential*? They look like poster children for state to me, but that might just be me, and I agree we may be discussing an edge case considering they are expressly written on user request (which would spell DATA); OTOH, the view directory is one of the state directories in need of regular purging (which would spell CACHE; CONFIG I find an unlikely choice either way).

To resume my misgivings, I think we should aim for this:

1. `&directory` default (swap files): `$XDG_CACHE_HOME/nvim/swap//`
2. `&backupdir` default:  `.,$XDG_CACHE_HOME/nvim/backup`
3. `&undodir` default: `$XDG_DATA_HOME/nvim/undo//` (no quibbles there)
4. ShaDa default: `$XDG_DATA_HOME/nvim/shada/main.shada` (ditto)
5. `&viewdir` default:  `$XDG_DATA_HOME/nvim/view` (or `$XDG_CACHE_HOME/nvim/view`)